openwisp_monitoring/device/static/monitoring/js/device-map.js

```diff
@@ -515,6 +514,10 @@
             netjsonGraph.eastWorldFeaturesAppended = true;
           }
         });
+
+        map.data.nodes.forEach(node => {
+            listenForLocationUpdates(map, node.id);
+        });
```

Comment:
Is this still needed? 

I remember we settled on using one single websocket listener.